assets/blocks/quiz/question-block/edit.js

```diff
@@ -0,0 +1,43 @@
+import { RichText, InnerBlocks } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Quiz question block editor.
+ *
+ * @param {Object}   props
```

Comment:
These were forgotten without a description.

Reply:
Fixed in 6c7e6ac